apps/dashboard/src/components/inbox-button.tsx

```diff
@@ -0,0 +1,27 @@
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/primitives/popover';
+import { APP_ID } from '@/config';
+import { Bell, Inbox, InboxContent } from '@novu/react';
+import { PopoverPortal } from '@radix-ui/react-popover';
+
+export const InboxButton = () => {
+  //   const { user } = useUser();
+
+  //   if (!user) {
+  //     return null;
+  //   }
+
+  return (
+    <Inbox subscriberId={'1234'} applicationIdentifier={APP_ID}>
```

Comment:
We will swap this out with the id

Reply:
Please convert this to an environment variables and use their presence as a feature flag to render <Inbox/> or not.